### ISSUE TITLE
MUI Accept behaviour on defining empty value

### ISSUE THREAD

**Issue by jens-maus** [amiga-mui/muidev#304]
## Summary

If I set the value of MUI Accept on a gadget to e.g. "0123456789-" to accept numbers only this works.  If I then set the value to "" - to clear the value it now accepts nothing at all, this is the opposite behaviour to MUI Reject.

## Expected results

If I set the value to "" the behaviour I would expect it to exhibit is that all characters are acceptable again.  If you look at MUI Reject give it a value = " !#$%*/" and then clear it with "", the gadget will no longer reject those characters - it resets.

In short I would like the behaviour of MUI Accept to be the same as that of MUI Reject - emptying the value resulting in default behaviour.
